azure-pipelines-master.yml

```diff
@@ -18,10 +17,9 @@ trigger:
 # GitHub creates a new ref when a pull request is created. The ref points to a 
 # merge commit, which is the merged code between the source and target branches 
 # of the pull request.
-pr:
-  branches:
-    include:
-    - master
+#
+# Opt out of pull request validation 
+pr: none
```

Comment:
Here we should definitely validate pull requests, i.e. it is absolutely necessary to run tests for all pull requests.

We cannot merge PRs to master if any tests have not been run. We have no way of knowing if they pass the tests otherwise.

Pull requests just do not need deployments in any environment.

Reply:
Ah, apparently this is done by a separate pipeline!